backend/backend_mantid_reducer.py

```diff
@@ -100,10 +190,17 @@ def filter_pulses(self, filter_pulses):
 
     @property
     def filter_interval_parameters(self):
-        return 'min,step,max'
```

Comment:
Even if returning the values is not implemented yet, providing the information that the requested format is `'min,step,max'` useful for the user.